internal/pkg/dependencies/common.go

```diff
@@ -169,6 +177,12 @@ func (v *CommonContainer) getServices() (storageapi.ServicesMap, error) {
 
 func (v *CommonContainer) getStorageIndex() (storageapi.Index, error) {
 	return v.storageApiIndex.InitAndGet(func() (*storageapi.Index, error) {
+		// Authentication is required
+		_, err := v.ProjectID()
+		if err != nil {
+			return nil, err
+		}
```

Comment:
Z indexu sa nacitavaju `features`, zaujimaju nas `features` k projektu, takze sa nemozu zoskat, ak nie sme prihlaseny, nie je nastavene `ProjectID` ... je to skor poistka, ak by bola chyba niekde inde.